config/config-feature-flags.yaml

```diff
@@ -88,11 +88,10 @@ data:
   # in the TaskRun/PipelineRun such as the source from where a remote Task/Pipeline
   # definition was fetched.
   enable-provenance-in-status: "false"
-  # Setting this flag to "full" to enable full embedding of `TaskRun` and `Run` statuses in the
-  # `PipelineRun` status. Set it to "minimal" to populate the `ChildReferences` field in the
+  # Setting this flag to "minimal" to populate the `ChildReferences` field in the
```

Comment:
I think we should keep the explanation of the different options for this flag and what they do